src/ToDoForm.jsx

```diff
@@ -1,53 +1,115 @@
 import React, { useState } from 'react';
+import { Button, TextField, Select, MenuItem, styled } from '@mui/material';
+import AddIcon from '@mui/icons-material/Add';
+import theme from './Theme';
+
+import ToDo from './ToDo';
 
 function ToDoForm({ onAddTodo }) {
     const [userInput, setUserInput] = useState({
         task: '',
         date: '1',
+        priority: ToDo.priorities.LOW,
     });
 
     const handleChange = (e) => {
```

Comment:
handleChange of what? i don't like `handleX` as namings for event handles. even though they appear a lot in tutorials - they are generally too vague and not descriptive. if it's only for updating a state, i prefer `on...`, like `onTextChange`, `onPriorityChange`, `onDateChange`